dist/images/ovnkube.sh

```diff
@@ -1476,6 +1482,7 @@ ovn-cluster-manager() {
     --logfile-maxsize=${ovnkube_logfile_maxsize} \
     --logfile-maxbackups=${ovnkube_logfile_maxbackups} \
     --logfile-maxage=${ovnkube_logfile_maxage} \
+    ${empty_lb_events_flag} \
```

Comment:
silly q: do nodes  still care for the `empty_lb_events_flag` after this change? I'm assuming yes and that is why you are not removing them.

Reply:
Yup, they still do. It is used to handle the main unidling logic